content/root/glossary.textile

```diff
@@ -33,6 +29,10 @@ h2(#apps). Apps
 
 An app (or application) in the context of Ably is a grouping of channels, queues and other resources that are managed as a collection. An app defines a namespace for its channels and queues so that they are unique to that application. An Ably account can contain multiple apps, with each app having its own set of API keys to limit client access to its resources. Access can then be further limited to a subset of channels and queues within an app and the permissions on those resources, such as whether a client can publish or subscribe to them.
 
+h2(#atm). ATM
+
+Active Traffic Management (ATM) is an Enterprise-only service that allows Ably engineers to manage the endpoints that customers connect to. This allows Ably to proactively route customer traffic to specific data centers or regions, ensuring service availability in the event of an incident.
```

Comment:
> Enterprise-only service that 

Didn't think of it before, but can we make this a link to  https://ably.com/pricing please.